eva_submission/eload_submission.py

```diff
@@ -44,14 +40,74 @@ def __init__(self, eload_number: int):
         for k in directory_structure:
             os.makedirs(self._get_dir(k), exist_ok=True)
 
+    def create_temp_output_directory(self):
```

Comment:
if this is truly temporary you could jus `tempfile.mkdtemp` https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

Reply:
I'm not convinced by this because, at least at the moment I want to control the deletion of the directory in case of failure.